<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetDialog</class>
 <widget class="QDialog" name="SetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>580</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>580</height>
    </rect>
   </property>
   <widget class="QWidget" name="containerWidget" native="true">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>58</y>
      <width>628</width>
      <height>468</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(40,40,40);</string>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QToolButton" name="generalBtn">
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>33</y>
       <width>74</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>General</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>628</width>
       <height>398</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QToolButton" name="saveBtn">
       <property name="geometry">
        <rect>
         <x>255</x>
         <y>297</y>
         <width>120</width>
         <height>36</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>72</x>
         <y>40</y>
         <width>230</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Wallet will be locked exceeding</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="languageComboBox">
       <property name="geometry">
        <rect>
         <x>245</x>
         <y>170</y>
         <width>109</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <item>
        <property name="text">
         <string>English</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>简体中文</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="nolockCheckBox">
       <property name="geometry">
        <rect>
         <x>477</x>
         <y>45</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Not lock</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>115</x>
         <y>120</y>
         <width>120</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Default vote setting</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="lockTimeSpinBox">
       <property name="geometry">
        <rect>
         <x>312</x>
         <y>40</y>
         <width>108</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>64</x>
         <y>170</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Preferred Language</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>427</x>
         <y>47</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>min</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="voteCheckBox">
       <property name="geometry">
        <rect>
         <x>255</x>
         <y>120</y>
         <width>261</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Transfer the default check vote</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="minimizeCheckBox">
       <property name="geometry">
        <rect>
         <x>255</x>
         <y>220</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Minimize the tray</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="closeCheckBox">
       <property name="geometry">
        <rect>
         <x>255</x>
         <y>250</y>
         <width>220</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Click the Close button to minimize</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QLineEdit" name="newPwdLineEdit">
       <property name="geometry">
        <rect>
         <x>225</x>
         <y>105</y>
         <width>195</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="confirmPwdLineEdit">
       <property name="geometry">
        <rect>
         <x>225</x>
         <y>162</y>
         <width>195</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="confirmBtn">
       <property name="geometry">
        <rect>
         <x>232</x>
         <y>238</y>
         <width>120</width>
         <height>36</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>167</y>
         <width>120</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>Verify password:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="tipLabel3">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>53</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>69</x>
         <y>53</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>Current password:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="tipLabel2">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>167</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>110</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>New password:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="tipLabel">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>110</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="oldPwdLineEdit">
       <property name="geometry">
        <rect>
         <x>225</x>
         <y>48</y>
         <width>195</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QToolButton" name="resyncBtn">
       <property name="geometry">
        <rect>
         <x>177</x>
         <y>170</y>
         <width>120</width>
         <height>36</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Resyn the blocks</string>
       </property>
      </widget>
      <widget class="QToolButton" name="scanBtn">
       <property name="geometry">
        <rect>
         <x>334</x>
         <y>170</y>
         <width>120</width>
         <height>36</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>scan</string>
       </property>
      </widget>
      <widget class="QLabel" name="linkLabel">
       <property name="geometry">
        <rect>
         <x>176</x>
         <y>100</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Official website:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="linkBtn">
       <property name="geometry">
        <rect>
         <x>297</x>
         <y>97</y>
         <width>120</width>
         <height>24</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;微软雅黑&quot;;
text-decoration: underline;
color: rgb(64, 154, 255);
background:transparent;</string>
       </property>
       <property name="text">
        <string>www.selfsell.com</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QToolButton" name="closeBtn">
     <property name="geometry">
      <rect>
       <x>594</x>
       <y>29</y>
       <width>9</width>
       <height>9</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="safeBtn">
     <property name="geometry">
      <rect>
       <x>98</x>
       <y>33</y>
       <width>74</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Safety</string>
     </property>
    </widget>
    <widget class="QToolButton" name="networkBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>33</y>
       <width>74</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>About</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>oldPwdLineEdit</tabstop>
  <tabstop>newPwdLineEdit</tabstop>
  <tabstop>confirmPwdLineEdit</tabstop>
  <tabstop>lockTimeSpinBox</tabstop>
  <tabstop>generalBtn</tabstop>
  <tabstop>networkBtn</tabstop>
  <tabstop>linkBtn</tabstop>
  <tabstop>safeBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
